js/index.js

```diff
@@ -0,0 +1,138 @@
+const featuredSpeakers = document.querySelector('.featured-speakers');
+const speakerSection = document.createElement('div');
+class Speakers {
+  constructor(name, title, about) {
+    this.name = name;
+    this.title = title;
+    this.about = about;
+  }
+}
+
+const speaker1 = new Speakers(
+  'Yochai Benkler',
+  'Founder & Managing Partner, Cowboy Ventures',
+  'Founding Partner at Cowboy Ventures, a team that backs seed-stage technology companies.',
+);
+
+const speaker2 = new Speakers(
+  'Kanyi Maqubela',
+  'Managing Partner, Kindred Ventures',
+  'Kanyi is a Managing Partner at Kindred Ventures, an early stage venture fund investing in mission-driven technology companies at the earliest stages.',
+);
+
+const speaker3 = new Speakers(
+  'RJ Scaringe',
+  'Founder & CEO, Rivian',
+  'RJ started Rivian to combine his love for cars with his desire to leave the world better for the generations who inherit it.',
+);
+
+const speaker4 = new Speakers(
+  'Serena Williams',
+  'Founding & Managing Partner, Serena Ventures',
+  'As a world-class athlete, businesswoman, philanthropist and mother, Serena Williams knows what it takes to win',
+);
+
+speakerSection.innerHTML = `
+    <h3 class="d-flex justify-content-center">Featured Speakers</h3>
+    <hr class="first">
+    <div class="featured-grid">
+        <div class="row mt-5 mt-lg-0">
+            <div class="col col-4">
+                <img src="/dist/images/Aileen-Lee.webp" alt="featured speaker" srcset="">
```

Comment:
- [ ] Kindly to stick to the original design, I would recommend you add the small square background to your image. 
### your design 👇️
![Screenshot from 2023-02-16 18-14-58](https://user-images.githubusercontent.com/89228743/219452629-90e584d9-779c-45a9-8ffc-9a5c6adbb121.png)

### the original design 👇️
![Screenshot from 2023-02-09 12-28-20](https://user-images.githubusercontent.com/89228743/217812916-6c850cad-ba84-4db5-936e-80b3c85779de.png)